cs/test/core/TypeAliasTests.cs

```diff
@@ -107,6 +108,130 @@ public static ArraySegment<byte> Convert(byte[] value, ArraySegment<byte> unused
         {
             return new ArraySegment<byte>(value);
         }
+
+        #region Box<T> for basic types
```

Comment:
Does anyone have ideas for how to reduce this boiler plate with today's Bond?

Today, the code that finds Convert methods doesn't find methods like `Convert<T>(Box<T> value, T unsued)`. This could be improved, but that's inappropriate for this commit. I need to leave now, but I've added a TODO to open an issue for this sort of improvement.